tools/internal_ci/linux/grpc_e2e_performance_v2.sh

```diff
@@ -78,7 +72,7 @@ buildConfigs() {
     shift 2
     tools/run_tests/performance/loadtest_config.py "$@" \
         -t ./tools/run_tests/performance/templates/loadtest_template_prebuilt_all_languages.yaml \
-        -s driver_pool="${DRIVER_POOL}" -s driver_image="${DRIVER_IMAGE}" \
+        -s driver_pool="${DRIVER_POOL}" -s driver_image= \
```

Comment:
when reading this, it's not really clear here why driver_image is being set to empty string

Reply:
It comes from the same place as worker images come from, when no image is specified. There is a set of images (driver image plus worker images for each language) that are built with each release of the test infrastructure and are used when no image is specified. That driver image uses the driver binary from the latest gRPC release, and specify only a run container; the worker images specify clone, build and run containers for the workers (in contrast to the ones used in continuous runs, which are prebuilt by the prepare_prebuilt_workers script and specify only a run container).